.github/workflows/node-zxcron-citr-extended-test-suite.yaml

```diff
@@ -0,0 +1,153 @@
+##
+# Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+
+name: CITR Extended CI Tests
+on:
+  schedule:
+    # Runs Extended Test Suite every three hours
+    - cron: '0 */3 * * *'
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  GRADLE_CACHE_USERNAME: ${{ secrets.gradle-cache-username }}
+  GRADLE_CACHE_PASSWORD: ${{ secrets.gradle-cache-password }}
+  GRADLE_EXEC: ionice -c 2 -n 2 nice -n 19 ./gradlew
+
+jobs:
+  extended-test-suite:
+    name: eXtended Test Suite
+    runs-on: network-node-linux-medium
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Expand Shallow Clone for and Spotless
+        if: ${{ !cancelled() }}
+        run: git fetch --unshallow --no-recurse-submodules
+
+      - name: Setup Java
+        uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4.0.0
+        with:
+          distribution: "temurin"
+          java-version: "21"
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@dbbdc275be76ac10734476cc723d82dfe7ec6eda # v3.4.2
+        with:
+          cache-read-only: false
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 20
+
+      - name: Compile
+        id: gradle-build
+        run: ${GRADLE_EXEC} assemble :test-clients:yahCliJar --scan
+
+      - name: Spotless Check
+        if: ${{ !cancelled() }}
+        run: ${GRADLE_EXEC} spotlessCheck --scan
```

Comment:
```suggestion
```

Should be removed. This would do nothing as it only checks the diff against _develop_, but it runs on _develop_ here.